lib/child-writer/writer-proxy.js

```diff
@@ -89,7 +89,7 @@ return permissions.isElevated().then((elevated) => {
     console.log(`Running: ${commandArguments.join(' ')}`);
 
     return permissions.elevateCommand(commandArguments, {
-      applicationName: packageJSON.build.productName,
+      applicationName: packageJSON.displayName,
```

Comment:
The issue is that this script doesn't run in the renderer process, so we don't have access to the DOM. I guess we could make the writer proxy take the title as a command line argument. I'll try to put a PR for it soon.